src/main/java/com/google/sps/servlets/MatchServlet.java

```diff
@@ -46,6 +46,6 @@ public synchronized void doPost(HttpServletRequest request, HttpServletResponse
       testRepository.addMatch(testUser, newMatch);
     }
 
-    response.sendRedirect("/index.html");
+    response.sendRedirect("/logged_in_homepage.html");
```

Comment:
Do we have tests for post requests to the match servlet?

Reply:
There are tests for the post request, it just doesn't test the redirect. Should I include something to test that, or is it enough to check manually on the page?

Reply:
In general the greater your test coverage, the more confidence you will have that any problems in your code will be caught before they affect users. As your system becomes more complex, manual testing becomes toilsome and I would suggest automating any manual testing where possible.

Reply:
That makes sense. I added a line in the MatchServletTest to verify that the redirect happens!